src/tactic/interactive_expr.lean

```diff
@@ -239,7 +251,8 @@ tc.stateless $ λ ft, do
     end,
   lcs ← local_context,
   lcs ← list.mfilter (filter_local ft) lcs,
-  lcs ← to_local_collection [] lcs,
+  lcs ← lcs.mmap $ to_local_collection,
+  let lcs := group_local_collection lcs,
   lchs ← lcs.mmap (λ lc, do
     lh ← local_c lc,
     ns ← pure $ lc.locals.map (λ n, h "span" [cn "goal-hyp b pr2"] [html.of_name $ expr.local_pp_name n]),
```

Comment:
`[cn "goal-hyp b pr2"]`. There should be a key here (my mistake) to stop react warning spam, It should be the unique name of `n`.

Reply:
Also on the next line `[cn "goal-hyp-type"]` -> `[cn "goal-hyp-type", key "type"]`